rust/transaction_parsing/src/parse_transaction.rs

```diff
@@ -664,7 +675,9 @@ pub fn entry_to_transactions_with_decoding(
                     .iter()
                     .find(|network| sign_display.network_name == network.specs.name)
                     .cloned()
-                    .unwrap();
+                    .ok_or_else(|| Error::HistoryUnknownNetwork {
```

Comment:
Does this happen when we attempt to open a history item for a tx that was signed with metadata proof?